source/E2E.Tests/Services/MobileParties/BanditPartyComponentTests.cs

```diff
@@ -72,4 +74,69 @@ public void ClientCreateParty_DoesNothing()
             Assert.False(client.ObjectManager.TryGetObject<MobileParty>(partyId, out var _));
         }
     }
+
+    [Fact]
+    public void ClientUpdateParty_DoesNothing()
+    {
+        // Arrange
+        var server = TestEnvironment.Server;
+        var client1 = TestEnvironment.Clients.First();
+
+        BanditPartyComponent component = GameObjectCreator.CreateInitializedObject<BanditPartyComponent>();
+        Hideout hideout = GameObjectCreator.CreateInitializedObject<Hideout>();
+        Settlement settlement = GameObjectCreator.CreateInitializedObject<Settlement>();
+
+        server.Call(() =>
+        {
+            component = GameObjectCreator.CreateInitializedObject<BanditPartyComponent>();
+            hideout = GameObjectCreator.CreateInitializedObject<Hideout>();
+            settlement = GameObjectCreator.CreateInitializedObject<Settlement>();
+        });
```

Comment:
```suggestion
        string componentId = TestEnvironment.CreateRegisteredObject<BanditPartyComponent>();
        string hideoutId = TestEnvironment.CreateRegisteredObject<Hideout>();
        string settlementId = TestEnvironment.CreateRegisteredObject<Settlement>();
```

**Then use the ids to resolve the objects in the clients scope**